src/client/visualizations/base-visualization/base-visualization.tsx

```diff
@@ -40,8 +40,11 @@ export class BaseVisualization<S extends BaseVisualizationState> extends React.C
   public static id = "base-visualization";
 
   // Way to get a static property without explicitly specifying the class
+  /**
+   * @deprecated
```

Comment:
remove it, `getBestVisualization` need to use something better and `className` on container should be from regular property.